content/version/2/1/code_of_conduct.gl.md

```diff
@@ -0,0 +1,133 @@
++++
+version = "2.1"
+aliases = ["/version/2/1"]
++++
+
+# Código de Conduta convido para contribuintes
+
+## O noso compromiso
+
+Nós, como membros, contribuintes, e responsables comprometémonos a facer da
+participación na nosa comunidade unha experiencia libre de acoso para todas as persoas, 
+independientemente da súa idade, tamaño corporal, discapacidade visible ou invisible,
+etnia, características sexuais, identidade e expresión de xénero, nivel de experiencia,
+educación, nivel socioeconómico, nacionalidade, aspecto físico, raza, casta, cor, relixión,
+identidade ou orientación sexual.
+
+Comprometémonos a actuar e interactuar dun modo que contribúa a unha comunidade aberta,
+acolledora, diversa, inclusiva e saudable.
+
+## Os nosos estándares
+
+Exemplos de comportamento que contribúen a un entorno positivo na nosa comunidade
+inclúen:
+
+* Demostrar empatía e amabilidade cara outras persoas
+* Respecto ante opinións, puntos de vista e experiencias diferentes 
+* Opinar de forma constructiva, e aceptar a crítica constructiva sen problemas
+* Asumir a responsabilidade e desculparse ante aqueles afectados polos nosos erros,
+  aprendendo de dita experiencia
+* Centrarse no que sexa millor non para nós coma individuos, senón para a comunidade coma conxunto.
+
+Exemplos de comportamentos que se consideran inaceptables inclúen: 
+
+* O uso de linguaxe ou imaxes sexualizadas, e aproximacións ou atencións sexuais de calquera tipo
+* _Troleo_, comentarios despectivos, insultantes ou derogatorios, e ataques políticos ou persoais
+* Acoso público ou privado
+* A publicación de información privada de outras persoas, coma por exemplo direccións físicas ou de correo electrónico,
+  sen o seu consentemento explícito 
+* Calquera outra conducta que puidera ser considerada inapropiada de xeito razonable nun entorno profesional
+
+## Aplicación das responsabilidades
+
+O liderado da comunidade é responsable de clarificar e garantir o cumplimento dos nosos estándares
+de comportamento aceptable, e tomará medidas correctivas con un alcance apropiado e xusto en resposta
+a calquera comportamento que se considere inapropiado, ameazante, ofensivo ou dañino.
+
+Os administradores da comunidade teñen o dereito e a responsabilidade de eliminar, editar, ou rexeitar 
+comentarios, _commits_, código, edicións de páxinas de documentación, tarefas, ou calquera outra contribución
+non alineada con este Código de Conduta, e comunicarán o razoamento detrás das decisións da moderación adoptadas
+cando sexa apropiado.
+
+## Alcance
+
+Este Código de Conduta aplica en tódolos espacios da comunidade, e tamén aplica
```

Comment:
A recomendación máis recente da RAG é que, aínda que na pronuncia oral digamos "tódolos", na escrita empreguemos "todos os".